iofog-agent-daemon/src/main/java/org/eclipse/iofog/field_agent/FieldAgent.java

```diff
@@ -1059,10 +1059,10 @@ public void start() {
             processRoutes(microservices);
         }
 
-        new Thread(pingController, "FieldAgent : Ping").start();
-        new Thread(getChangesList, "FieldAgent : GetChangesList").start();
-        new Thread(postStatus, "FieldAgent : PostStatus").start();
-        new Thread(postDiagnostics, "FieldAgent : PostDiagnostics").start();
+        new Thread(pingController, "FAPC").start();
```

Comment:
Please define these abbreviations as constants in a separate file

Reply:
ok